src/Ng/ElfieFromCatalogCollector.cs

```diff
@@ -412,9 +412,9 @@ void StageLocalPackageContents(string localPackageFile, string tempDirectory)
         {
             Trace.TraceInformation("#StartActivity StageLocalPackageContents");
 
-            // The assemblies need to reside in a lib subdirectory. This is where the assembly
+            // The reference assemblies need to reside in a ref subdirectory. This is where the assembly
             // references are in a real package.
-            string libDirectory = Path.Combine(tempDirectory, "lib");
+            string libDirectory = Path.Combine(tempDirectory, "ref");
```

Comment:
Yes, I will make that change. BTW, this was essentially the root cause of the missing system.threading.thread. The type definitions were in the ref directory and we were indexing the lib directory. This is part of the reason I'd prefer if the indexer was responsible for the validation of the decompressed files. Our current thinking is that we might have to index the entire contents of the package, not just the lib and ref directories. So I would rather not put an artificial restriction in the crawler yet.